index.html

```diff
@@ -2612,6 +2617,102 @@ <h3 id="applying">
+          meta keywords, catalogs and stores are not required to honor this hint.
+        </p>
+        <p>
+          The <dfn>steps for processing the <code>categories</code>
+          member</dfn> are given by the following algorithm. The algorithm
+          takes a <var>manifest</var> as an argument. This algorithm returns an
+          array of strings or <code>undefined</code>.
+        </p>
+        <ol>
+          <li>Let <var>categories</var> be an empty list.</li>
+          <li>Let <var>unprocessed categories</var> be the result of calling
+          the <a>[[\GetOwnProperty]]</a> internal method of <var>manifest</var>
+          with argument "<code>categories</code>".</li>
+          <li>If <var>unprocessed categories</var>is an array, then:
+            <ol>
+              <li>For each <var>potential category</var> in the array:
+                <ol>
+                  <li>
+                    <a>Trim</a>(<var>value</var>) and convert 
+                    <a>to ASCII lowercase</a>
+                  </li>
+                  <li>
+                    Append <var>potential category</var> to 
+                    <var>categories</var>
+                  </li>
+                </ol>
+              </li>
+            </ol>
+          </li>
+          <li>
+            Otherwise, if <var>unprocessed categories</var> is not 
+            <code>undefined</code>
+            <ol>
+              <li>
+                <a>Issue a developer warning</a> that the type is not 
+                supported.
+              </li>
+            </ol>
+          </li>
+          <li>Return <var>categories</var></li>
+        </ol>
+        <p>
+          The categories string array is case insensitive and converted 
+          to lower-case by following the processing algorithm. Thus, 
+          <code>sports</code>, <code>Sports</code>, <code>SPORTS</code>, 
+          and <code>SpOrTs</code> are all equivalent.<br />
+          NOTE: Manifest authors are encouraged to use lower-case.
+        </p>
+        <p>
+          Manifest authors SHOULD use the below common categories:
+        </p>
+        <ul>
+          <li>books</li>
+          <li>business</li>
+          <li>education</li>
+          <li>entertainment</li>
+          <li>finance</li>
+          <li>fitness</li>
+          <li>food</li>
+          <li>games</li>
+          <li>government</li>
+          <li>health</li>
+          <li>kids</li>
+          <li>lifestyle</li>
+          <li>magazines</li>
+          <li>medical</li>
+          <li>music</li>
+          <li>navigation</li>
+          <li>news</li>
+          <li>personalization</li>
+          <li>photo</li>
+          <li>politics</li
+          <li>productivity</li>
+          <li>security</li>
+          <li>shopping</li>
+          <li>social</li>
+          <li>sports</li>
+          <li>travel</li>
+          <li>utilities</li>
+          <li>weather</li>
```

Comment:
I am fine with this list in the spec, but I think we should create a wiki like https://github.com/w3c/manifest/wiki/Extensions-Registry maybe https://github.com/w3c/manifest/wiki/Categories that replicates the list and where we can add new ones after a published stable spec, as they become available.

We would of course have to link to that wiki entry in the spec with appropriate text.

Reply:
OK, I created this page, feel free to edit

Reply:
@RobDolinMS could you add some text and link to this?

Reply:
Thanks @kenchris.  I've just added text (lines 2680 - 2686) mirroring the language used for the platform member.

Do you think I should also delete the list of common categories values from the spec (since it's in the wiki page?)

Reply:
I think they should be removed, to be honest.